src/main/java/com/checkout/payments/contexts/PaymentContexts.java

```diff
@@ -16,36 +16,36 @@
 @NoArgsConstructor
 public class PaymentContexts {
 
-    public Long amount;
+    private Long amount;
 
-    public Currency currency;
+    private Currency currency;
 
     @SerializedName("payment_type")
-    public PaymentType paymentType;
+    private PaymentType paymentType;
 
     @SerializedName("authorization_type")
     private String authorizationType;
 
-    public boolean capture;
+    private Boolean capture;
```

Comment:
I don't know enough about Java to know the difference between Boolean and boolean, but is does this change of types risk breaking something?

Reply:
Bolean is better, no problem about it, because the Boolean class wraps a value of the primitive type boolean in an object. An object of type Boolean contains a single field whose type is boolean.